Harvest.Web/ClientApp/src/Util/DateHelper.ts

```diff
@@ -0,0 +1,7 @@
+export const EnsureUTC = (date: Date) => {
```

Comment:
Nitpicky but do the function camel case, that's what we do in other places. 'ensureUTC()'